pkg/terminal/command.go

```diff
@@ -1216,6 +1216,8 @@ func edit(t *Term, ctx callContext, args string) error {
 	}
 
 	cmd := exec.Command(editor, fmt.Sprintf("+%d", lineno), file)
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
```

Comment:
let's do stderr to: `cmd.Stderr = os.Stderr`.

Reply:
ok